docs/design/baremetal/networking-infrastructure.md

```diff
@@ -32,19 +38,36 @@ The API VIP first resides on the bootstrap VM. The `keepalived`
 instance here is managed by systemd and a script is used to generate
 the `keepalived` configuration before launching the service using
 `podman`. See [here](../../../data/data/bootstrap/baremetal/README.md)
-for more informations about the relevant bootstrap assets.
+for more information about the relevant bootstrap assets.
 
-Once the control plane machines come up, the VIP will move to the one
-of these machines. This happens because the `keepalived` instances on
-control plane machines are configured (in `keepalived.conf`) with a
-higher
+The VIP will move to one of the control plane nodes, but only after the
+bootstrap process has completed and the bootstrap VM is stopped. This happens
+because the `keepalived` instances on control plane machines are configured (in
+`keepalived.conf`) with a lower
 [VRRP](https://en.wikipedia.org/wiki/Virtual_Router_Redundancy_Protocol)
-priority. These `keepalived` instances are run as [static
-pods](https://kubernetes.io/docs/tasks/administer-cluster/static-pod/)
-and the relevant assets are [rendered by the Machine Config
+priority. This ensures that the API on the control plane nodes is fully
+functional before the API VIP moves.
```

Comment:
@markmc you raised the discrepency about VRRP priorities out of band.  I have corrected the documentation to reflect what's happening here.  FYI